### PR TITLE
Karma: proposal to simplify mouse events

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: assets/src/edit-story/elements/text/karma/edit.karma.js [hunk 1]
```diff
@@ -101,12 +101,10 @@ describe('TextEdit integration', () => {
         expect(boldToggle.checked).toEqual(true);
 
         // Exit edit mode by clicking right outside the editor.
-        // @todo: this API is a bit low level. would be good to add additional
-        // ways to do events relative to targets and each other. Maybe something
-        // like `move({relative: element, anchor: 'topleft', x: -10})`?
-        const { x, y } = editor.getBoundingClientRect();
-        await fixture.events.mouse.move(x - 10, y);
-        await fixture.events.mouse.down();
+        await fixture.events.mouse.seq(({ moveRel, down }) => [
```
Comment (dvoytenko) [inline assets/src/edit-story/elements/text/karma/edit.karma.js:104]: @miina This is a replacement I'm thinking about to simplify the event sequences. WDYT?

**Comment by dvoytenko** [comment]
@barklund @miina PTAL.

File: assets/src/edit-story/karma/fixtureEvents.js [hunk 2]
```diff
@@ -227,6 +243,70 @@ class Mouse {
...
+    this._xy = [0, 0];
+
+    const relToDelta = (rel, size) => {
+      if (!rel) {
+        return 0;
+      }
+      if (typeof rel === 'number') {
+        return rel;
+      }
+      if (rel.endsWidth('%')) {
```
Comment (merapi) [inline assets/src/edit-story/karma/fixtureEvents.js]:
```suggestion
      if (rel.endsWith('%')) {
```

**Review by miina — APPROVED** [review]
LGTM, let's try it out to see how it feels.

File: assets/src/edit-story/karma/fixtureEvents.js (hunk 2 above)
Comment (dvoytenko) [reply assets/src/edit-story/karma/fixtureEvents.js]: Thanks!

**Pull request merged**